### PR TITLE
Add rural payments data source and corresponding resolver for customer and business endpoints

### PR DESCRIPTION
**Pull request by dtaylor7**
- Add rural payments data source & tests
 - Add types, resolver, transformer and tests for person
 - Add types, resolver, transformer and tests for business
 - Refactor mock server to work with tests

### CODE REVIEW

File: app/graphql/types/customer/customer.gql [hunk 1]
```diff
@@ -28,7 +28,7 @@ type CustomerAuthenticationQuestions {
 }
 
 type Customer {
-  referenceNumber: ID!
```
Comment (jack-mccabe-defra) [inline app/graphql/types/customer/customer.gql:31]: Is there a reason we changed this? We may encounter other customer ids than CRN so maybe best to leave this as is?

Comment (dtaylor7) [reply app/graphql/types/customer/customer.gql:31]: so actually ID is not the crn, there is a crn inside the customer info

File: docker-compose.test.yaml [hunk 2]
```diff
@@ -10,9 +10,11 @@ services:
...
     volumes:
       - ./app:/home/node/app
+      - ./mocks:/home/node/mocks
       - ./test:/home/node/test
       - ./test-output:/home/node/test-output
       - ./jest.config.js:/home/node/jest.config.js
       - ./package.json:/home/node/package.json
     environment:
       RURAL_PAYMENTS_AGENCY_LAND_API_URL: https://environment.data.gov.uk
+      RURAL_PAYMENTS_API_URL: http://localhost:3100/rpp/
```
Comment (jack-mccabe-defra) [inline docker-compose.test.yaml:20]: automated tests are run using this file, do you need to start the mock server as well?

Comment (dtaylor7) [reply docker-compose.test.yaml:20]: yes it does start when the tests do

**Pull request merged**